Security-Hub/cloudformation/README.md

```diff
@@ -0,0 +1,35 @@
+![CrowdStrike Falcon](https://raw.githubusercontent.com/CrowdStrike/falconpy/main/docs/asset/cs-logo.png)
+![Twitter URL](https://img.shields.io/twitter/url?label=Follow%20%40CrowdStrike&style=social&url=https%3A%2F%2Ftwitter.com%2FCrowdStrike)
+
+# CloudFormation deployment example
+This example CFT demonstrates deployment of the following:
++ A single VPC with a single subnet and an internet gateway. VPC Flow logging is enabled.
+This VPC resides in __us-west-2__ (this can be changed).
++ A __t2.micro__ AWS Linux 2 instance that pre-installs the Security Hub integration application as a bootstrap.
+    - This instance is assigned an elastic IP. This is not a requirement and only included for demo purposes.
++ A SQS queue, dead-letter queue and Lambda trigger for handling detections identified as findings.
++ A Lambda handler that consumes detections posted to the SQS queue and submits findings to AWS Security Hub based.
++ Security Hub is activated for the region deployed.
+
+> Please note: You will need to manual subscribe to CrowdStrike findings in the region after deploying this template.
+
+## Prerequisites
++ Valid CrowdStrike Falcon API credentials allowing access _READ_ to Event Streams, Hosts, and Detections.
```

Comment:
No more Host Downloads for MSSP?

Reply:
Let me check the code. When I typed this I thought it only required READ...

Reply:
One call to GetDeviceDetails, so I think we're ok.